drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -186,7 +183,7 @@ def studio_view(self, context):
 
         js_templates = loader.load_unicode('/templates/html/js_templates.html')
         help_texts = {
-            field_name: self._(field.help)
+            field_name: self.ugettext(field.help)
```

Comment:
@itsjeyd Yes currently I was looking into that. Actually I was running this command 
`django-admin makemessages --locale=en --extension .html` to extract string from HTML files, It is working fine but not extracting string from `js_templates.html`. It seems me that, `js_templates.html` is using `handlebars {{i18n}}` tags.